pkgs/applications/office/p3x-onenote/default.nix

```
@@ -0,0 +1,57 @@
+{ stdenv, appimageTools, autoPatchelfHook, desktop-file-utils
+  , fetchurl, runtimeShell }:
```

jnit: comma usually start at the begging of the line for input attr sets
```suggestion
, fetchurl, runtimeShell }:
```

---

thanks, I've upped it since runtimeShell is now removed